### DISCUSSION TITLE
Skyline crashes

### DISCUSSION THREAD

**Discussion by ghost** [skyline-emu/skyline#331] [Q&A] [answered]
In Samsung galaxy A12 runnign android 11

**Comment by ghost** [comment]
It crashes when runniing a game and doesnt run

**Comment by ghost** [comment]
The log is https://drive.google.com/file/d/1D9YNk_jiIFuHJZENyHzp1oJ2kDIt8Vwb/view?usp=drivesdk

**Comment by nickbeth** [comment, marked as answer]
You've already created 2 Issues that have been properly replied to. Your device will never be supported because it doesn't satisfy the minimum requirements. Please refrain from opening new Issues or Discussions, or you will be blocked.

**Comment by iniMerchant64** [comment]
Just stop. Your SoC doesnt support the required things on hardware level meaning software wont fix it.

**Comment by ghost** [comment]
If so why it runs perfectly without sound on this 
https://github.com/skyline-emu/skyline/actions/runs/1471381086

**Comment by ghost** [comment]
Which means i can emulate even without full speed!

**Reply by iniMerchant64** [reply]
I explained before. That was before any of the vulkan part of gpu was implemented. Your phone does not support the required vulkan fuctions to work here. Logs tell that

**Comment by ghost** [comment]
Well It is possible to remove vulkan and use an alternative gpu such as opengl that device support.the device is capable of rubning at a playable speed yet vulkan is unsupported so

**Reply by iniMerchant64** [reply]
Skyline will never use ogl due to how inferior it is to vulkan. It will **never** have any other backend

**Comment by ghost** [comment]
In that case i dont think the features.shaderImageGatherExtended feature is important there must be an alternative to that feature or a way to use software gpu.
I can also use gltools with root and/or patch the drivers by crafting a magisk module (i have root)

**Comment by ghost** [comment]
I dont mind inferior display quality.I only care about running it at all

**Reply by iniMerchant64** [reply]
Listen, devs will not go out of their way just to implement ogl especially for you and make skyline work on your incredibly underpowered phone.
